### ISSUE TITLE
AnonCreds RS should take predicates into account when querying credentials for a request

### ISSUE THREAD

**Issue by TimoGlastra** [openwallet-foundation/agent-framework-javascript#1685]
I think it will currently return e.g. a credential where age = 18 even if the predicate is age >= 21.

I couldn't find code that correctly does a post filter on the found credentials. @genaris is that correct?

@genaris

**Comment by genaris** [comment]
Yes, I'm afraid the post filtering for predicates in holder service has not been implemented yet.
